<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="lineEdit_Inforcn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SCP">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Connect Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_COM">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCOM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM PORT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BAUD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblBAUD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BAUD RATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA SIZE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DTS">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTen">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DUMMY ECUS FOR TESTING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CANID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCANID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_HEX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SentHEX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sent_HEX</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>711</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Sent">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DATASent">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA Sent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Table Status Sent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisConnect">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DisConnect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Receip">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCANID_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CYCLE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CYCLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectRx">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ConnectRx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnectRx">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DisconnectRx</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_COM_RX">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearSent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearSent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearReceive">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearReceive</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCOMRx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COM_PORT_RX</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Receive">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DATASent_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA Received</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Table Status Receive</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
